libraries/test_helpers.rb

```diff
@@ -0,0 +1,12 @@
+class Mock
+  class ShellCommandResult
+    @stdout = ''
+    def initialize(stdout)
+      @stdout = stdout
+    end
+
+    def stdout
+      @stdout
+    end
```

Comment:
CI linter suggests doing
```suggestion
     attr_accessor :stdout
```
I think